imports/plugins/included/connectors-shopify/server/hooks/orders.js

```diff
@@ -0,0 +1,278 @@
+}
+
+/**
+ * @private
+ * @summary Convert address object into Shopify-compatible format
+ * @param {Object} address - the address to convert
+ * @returns {Object} converted address
+ */
+function convertAddress(address) {
+  const convertedAddress = {};
+  convertedAddress.address1 = address.address1;
+  convertedAddress.address2 = address.address2 || "";
+  convertedAddress.city = address.city;
+  convertedAddress.country = address.country;
+  convertedAddress.country_code = address.country;
+  convertedAddress.name = address.fullName;
+  const [ firstName, ...lastName ] = address.fullName.split(" ");
+  convertedAddress.first_name = firstName;
+  convertedAddress.last_name = lastName.join(" ");
+  convertedAddress.phone = address.phone;
+  convertedAddress.zip = address.postal;
+  convertedAddress.province_code = address.region;
+  return convertedAddress;
+}
+
+/**
+ * @private
+ * @summary Create a customer object in Shopify-compatible format
+ * @param {Object} address - Converted address object
+ * @param {Object} order - Original order we are converting
+ * @returns {Object} Shopify-compatible customer object
+ */
+function convertCustomer(address, order) {
+  let phone;
+  if (address.country_code === "US") {
+    phone = `+1${address.phone}`; // shopify wants this corny +1 in front of the phone
+  } else {
+    phone = address.phone;
+  }
+  const customer = {
+    accepts_marketing: false,
+    email: order.email,
+    phone,
+    first_name: address.first_name,
+    last_name: address.last_name
+  };
+  return customer;
+}
+
+/**
+ * @summary Export an order to Shopify
+ * @param {Object} doc - The order to convert
+ * @returns {Promise.<Array>} - An array of exported orders
+ */
+export async function exportToShopify(doc) {
+  const numShopOrders = doc.billing.length; // if we have multiple billing, we have multiple shops
+  Logger.debug(`Exporting ${numShopOrders} order(s) to Shopify`);
+  const shopifyOrders = [];
+  for (let index = 0; index < numShopOrders; index++) {
+    // send a shopify order once for each merchant order
+    const shopId = doc.billing[index].shopId;
+    const shopifyOrder = convertRcOrderToShopifyOrder(doc, index, shopId);
+    Logger.debug("sending shopify order", shopifyOrder, doc._id);
+    const apiCreds = getApiInfo(shopId);
+    const shopify = new Shopify(apiCreds);
+    const newShopifyOrder = await shopify.order.create(shopifyOrder);
+    markExported(newShopifyOrder, shopId, doc);
+    shopifyOrders.push(newShopifyOrder);
+  }
+  return shopifyOrders;
+}
+
+
+Orders.after.insert((userId, doc) => {
+  const pkgData = Reaction.getPackageSettings("reaction-connectors-shopify");
+  if (pkgData) {
+    const { settings } = pkgData;
+    const { synchooks } = settings;
+    if (synchooks) {
+      synchooks.forEach((hook) => {
+        if (hook.topic === "orders" && hook.event === "orders/create") {
+          if (hook.syncType === "exportToShopify") { // should this just be dynamic?
```

Comment:
I think it should be dynamic. Right now it seems like the idea of a synchook exists in theory, but you'd still need to add code to this hook to add a different hook.